README.md

```diff
@@ -148,7 +148,7 @@ openldap::server::overlay { 'memberof on dc=example,dc=com':
 ### Configuring ACPs/ACLs
 
 [Documentation](http://www.openldap.org/devel/admin/slapdconf2.html) about olcAcces state the following spec:
-> 5.2.5.2. olcAccess: to &lt;what&gt; [ by &lt;who&gt; [&lt;accesslevel&gt;] [&lt;control&gt;] ]+
+> 5.2.5.2. olcAccess: to &lt;what&gt; \[ by &lt;who&gt; \[&lt;accesslevel&gt;\] \[&lt;control&gt;\] \]+
```

Comment:
The only purpose of this is to fix syntax highlighting in my editor.  I guess it is confused by the way `[ref]` can be used to mark links reference and `[ref]: url` to associate the url with the actual ref.